### ISSUE TITLE
[Accessibility] Incorrect use of image for text

### ISSUE THREAD

**Issue by amberhinds** [woocommerce/woocommerce-gateway-amazon-pay#328]
## Audit Information
This issue came from an audit WooCommerce had Equalize Digital complete in December 2025. I'm moving the issue out of Linear (WooA11Y-414) to this repo.

- EQD Audit ID: WooExt-509
- Component/Page: Single Products
- Affected Element: Use your Amazon account
- URL where issue was discovered: https://apay-woo.mystagingwebsite.com/product/lightweight-wool-table/

## WCAG Compliance

- Success Criterion: 1.4.5 Images of Text
- WCAG Version: 2
- Level: AA
- Severity: High

## Issue Description

Text is presented as an image, and there is no alternative accessible format provided. As a result, the text cannot be resized or styled by assistive technologies, and it is not translatable by browser or website translation tools. The image also has an aria-label that does not match the text in the image.

### Impacted Populations

People with low vision using screen magnifiers, Screen reader users (blind/low vision and neurodivergent people), People relying on translation tools

## Recommended Fix

Replace the image containing text with actual text in an accessible format, such as HTML text in a paragraph. If the use of an image for text is unavoidable, ensure that the text is provided in a separate, accessible format that can be resized, styled, and read by assistive technologies. The alternative text or accessible name must always match the text in the image exactly.

## Code Example

`<img src="https://static-na.payments-amazon.com/assets/maxo/microtext/en_US.svg" aria-label="Use your Amazon Pay Sandbox test account">`

## Screenshots

https://drive.google.com/file/d/1kv6hT3kyMqDOsVT3o4Noo-f23Jli_VjL/view?usp=drivesdk

**Comment by ChristianAF** [comment]
Hey @amberhinds, this issue is not valid for the plugin. It's related to Amazon Pay SDK. we can not handle the rendeting of the button.
